source/operations/network-encryption.rst

```diff
@@ -10,256 +10,330 @@ Network Encryption (TLS)
+   For example, the following command generates a self-signed certificate with a set of IP and DNS SANs associated to the MinIO Server hosts:
 
-Supported TLS Cipher Suites
----------------------------
+   .. code-block:: shell
+
+      certgen -host "localhost,minio-*.example.net"
+
+   You can place the generated ``public.crt`` and ``private.key`` into the ``/.minio/certs`` directory to enable TLS for the MinIO deployment.
+   Applications can use the ``public.crt`` as a trusted Certificate Authority to allow connections to the MinIO deployment without disabling certificate validation.
+
+.. cond:: windows
+
+   The MinIO server searches the following directory for TLS keys and certificates:
+
+   .. code-block:: shell
+
+      %%USERPROFILE%%\.minio\certs 
+
+   For deployments started with a custom TLS directory :mc-cmd:`minio server --certs-dir`, use that directory instead of the defaults.
+
+   Place the TLS certificates for the default domain (e.g. ``minio.example.net``) in the ``/certs`` directory, with the private key as ``private.key`` and public certificate as ``public.crt``.
+
+   For example:
+
+   .. code-block:: shell
+      
+      %%USERPROFILE%%\.minio\certs
+        private.key
+        public.crt
+
+   Where ``%%USERPROFILE%%`` is the location of the `User Profile folder <https://docs.microsoft.com/en-us/windows/deployment/usmt/usmt-recognized-environment-variables>`__ of the user running the MinIO Server process.
+
+   You can use the MinIO :minio-git:`certgen <certgen>` to mint self-signed certificates for enabling TLS for evaluating MinIO with TLS enabled.
+   For example, the following command generates a self-signed certificate with a set of IP and DNS SANs associated to the MinIO Server hosts:
+
+   .. code-block:: shell
+
+      certgen.exe -host "localhost,minio-*.example.net"
+
+   You can place the generated ``public.crt`` and ``private.key`` into the ``\.minio\certs`` directory to enable TLS for the MinIO deployment.
+   Applications can use the ``public.crt`` as a trusted Certificate Authority to allow connections to the MinIO deployment without disabling certificate validation.
+
+
+Multiple Domain-Based TLS Certificates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. cond:: k8s
+
+   The MinIO Operator supports attaching user-specified TLS certificates when :ref:`deploying <minio-k8s-deploy-minio-tenant-security>` or :ref:`modifying <minio-k8s-modify-minio-tenant-security>` the MinIO Tenant.
+
+   These custom certificates support `Server Name Indication (SNI) <https://en.wikipedia.org/wiki/Server_Name_Indication>`__, where the MinIO server identifies which certificate to use based on the hostname specified by the connecting client.
+   For example, you can generate certificates signed by your organizations preferred Certificate Authority and attach those to the MinIO Tenant.
```

Comment:
This is also the first time that you've used "Certificate Authority" on the page instead of just CA. We need to define CA  earlier and then just use that here. Maybe making more use of :abbr:`CA <Certificate Authority>`.